themes/yamanpour.py

```diff
@@ -0,0 +1,51 @@
+class DefaultColor:
```

Comment:
You should not create a class called `DefaultColor`. You should instead just make this `class Color(DefaultColor)` and remove the second class in this file.